bedrock/firefox/views.py

```diff
@@ -484,13 +500,12 @@ def new(request):
         variant = None
 
     if scene == '2':
-        # `wait-face`, `reggiewatts` variations are currently localized for both en-US and de locales.
-        if lang_file_is_active('firefox/new/wait-face', locale) and experience == 'waitface':
-            template = 'firefox/new/wait-face/scene2.html'
-        elif lang_file_is_active('firefox/new/reggiewatts', locale) and experience == 'reggiewatts':
-            template = 'firefox/new/reggie-watts/scene2.html'
-        else:
-            template = 'firefox/new/scene2.html'
+        # send to new permanent scene2 URL (bug 1438302)
+        thanks_url = reverse('firefox.download.thanks')
+        query_string = request.META.get('QUERY_STRING', '')
+        if query_string:
+            done_url = '?'.join([thanks_url, force_text(query_string, errors='ignore')])
```

Comment:
Any reason to keep the `scene=2` query param around? Maybe for keeping track of how many folks are still hitting the old URL?

Reply:
I thought maybe it'd be useful for that, yes, but also I didn't want to go to the trouble to remove it since it's not hurting anything. I'm happy to though if we'd prefer it that way.